src/components/MainContact/test

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useEffect } from 'react';
+import { useInView } from 'react-intersection-observer';
+import { useSpring, animated } from 'react-spring';
+import cn from 'classnames';
+import { charityAPI } from '../../clients';
+import Loader from './ContentLoader';
+import './styles.scss';
+
+const MainContactContainer = () =>{
+  
+}
+const MainContact = () => {
+  const [data, setData] = useState({});
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const getData = () => {
+    setLoading(true);
+    charityAPI('/main-contacts')
+      .then(({ data }) => {
+        setData(data);
+        setLoading(false);
+        setError(false);
+      })
+
+      .catch(error => {
+        setLoading(false);
+        setError(true);
+        setErrorMessage("Couldn't fetch data");
+      });
+  };
+
+  useEffect(() => {
+    setLoading(true);
+    getData();
+  }, []);
+  //Scroll observation
+  const [ref, inView] = useInView({
+    threshold: 0.3,
+    triggerOnce: true
+  });
+  const fade = useSpring({
+    opacity: inView ? 1 : 0
+  });
+  if (loading) {
+    return (
+      <div className="flex justify-center items-center">
+        <div className="flex  mx-8">
+          <Loader />
+        </div>
+        <div className="flex mx-8">
+          <Loader />
+        </div>
+        <div className="flex  mx-8">
+          <Loader />
+        </div>
+      </div>
+    );
+  } else if (error) {
+    return (
+      <div>
+        {errorMessage},{' '}
+        <a href="#/" onClick={getData} className="text-c200">
+          retry?
+        </a>
+      </div>
+    );
+  } else {
+    return (
+      <animated.div style={fade}>
+        <div ref={ref} className="contact-info flex items-center justify-end">
+          {data.map(({ _id, title, sub_title, icon: { url } }, index) => {
+            const isLast = index === data.length - 1;
+
+            return (
+              <div
+                className={cn(
+                  'contact-info--details flex items-center px-5 lg:px-10',
+                  {
+                    'border-right-header': !isLast,
+                    'pr-0': isLast
+                  }
+                )}
+                key={_id}
```

Comment:
Let us use `id` instead of `_id` in lines 73 and 85 so that it is consistent with other components